Source/Libraries/CorruptCore/Stockpile/SaveException.cs

```diff
@@ -0,0 +1,25 @@
+namespace RTCV.CorruptCore.StockpileNS
```

Comment:
Not a huge fan of this namespace. Maybe we could pull exceptions into a common namespace?

Reply:
Agreed, I don't love `RTCV.CorruptCore.StockpileNS`. I could create `RTCV.CorruptCore.Exceptions` (with a `CorruptCore\Exceptions` folder) and put it there.

This is the only custom exception that I could find in `CorruptCore`. However, I have a feeling there may be more on the way (like a `LoadException`)

Reply:
Done in the latest commit

Reply:
Sounds good. I do agree that more custom exceptions will make sense which is why I brought this up